community.md

```diff
@@ -63,9 +63,9 @@ Subscribe to our [mailing list](https://mail.mozilla.org/listinfo/dev-addons) de
 
 ### Questions about add-on development
 
-You can get help and support by posting to [Mozilla's community forum](https://discourse.mozilla.org/c/add-ons), [Matrix Chat Rooms](https://wiki.mozilla.org/Matrix): [#add-ons](https://matrix.to/#/#add-ons:mozilla.org), [#addons-reviewers](https://matrix.to/#/#addon-reviewers:mozilla.org), [Stack Overflow](http://stackoverflow.com/questions/tagged/firefox-addon), or the [developer mailing list](https://mail.mozilla.org/listinfo/dev-addons).
+You can get help and support by posting to [Mozilla's community forum](https://discourse.mozilla.org/c/add-ons), [Matrix Chat Rooms](https://wiki.mozilla.org/Matrix): [#add-ons](https://bit.ly/2PKCHuH), [#addons-reviewers](https://bit.ly/32Odp4i), [Stack Overflow](http://stackoverflow.com/questions/tagged/firefox-addon), or the [developer mailing list](https://mail.mozilla.org/listinfo/dev-addons).
```

Comment:
```suggestion
You can get help and support by posting to [Mozilla's community forum](https://discourse.mozilla.org/c/add-ons), [Matrix Chat Rooms](https://wiki.mozilla.org/Matrix): [#add-ons](https://mzl.la/2u8ZGbg), [#addons-reviewers](https://bit.ly/32Odp4i), [Stack Overflow](http://stackoverflow.com/questions/tagged/firefox-addon), or the [developer mailing list](https://mail.mozilla.org/listinfo/dev-addons).
```

Reply:
Let's break this up a little bit. It should read: 

You can get help and support by posting to [Mozilla's community forum](https://discourse.mozilla.org/c/add-ons), [Stack Overflow](http://stackoverflow.com/questions/tagged/firefox-addon), or the [developer mailing list](https://mail.mozilla.org/listinfo/dev-addons). You can also talk with us on [Matrix](https://wiki.mozilla.org/Matrix) in the [Add-ons room](https://mzl.la/2u8ZGbg) (for general development questions) or in the [add-on reviewers room](https://bit.ly/32Odp4i) (for questions about an add-on review).